ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/Entry/SpeckleMappingsCommandWin.cs

```diff
@@ -28,6 +29,8 @@ protected override Result RunCommand(RhinoDoc doc, RunMode mode)
     }
     catch (Exception e)
     {
+      // needs more investigation. logging to seq for now.
```

Comment:
Similarly we need to consider use of IsFatal() here but it might fall under the remit of that toplevel exception handling issue, I will raise one and ping the connectors team and then we can be referncing these places with the ticekt and disabling the warning for now.

Reply:
yup - another toplevel exception